### PR TITLE
docs(devServer): add `writeToDisk` option

### PR DESCRIPTION
**Pull request by AlexDubok**
docs(devServer): add `writeToDisk` option

### CODE REVIEW

File: src/content/configuration/dev-server.md [hunk 1]
```diff
@@ -1226,3 +1226,36 @@ module.exports = {
 If this is too heavy on the file system, you can change this to an integer to set the interval in milliseconds.
 
 See [WatchOptions](/configuration/watch/) for more options.
+
+
+## `devServer.writeToDisk` 🔑
+
+`boolean` `function`
```
Comment (EugeneHlushko) [inline src/content/configuration/dev-server.md]: please add default value here and remove below. please add function signature here e.g. `function (param1, param2)`

File: src/content/configuration/dev-server.md [hunk 2]
```diff
@@ -1226,3 +1226,36 @@ module.exports = {
...
+
+Default: `false`. Tell dev-server to write generated assets to the disk.
```
Comment (EugeneHlushko) [inline src/content/configuration/dev-server.md]:
```suggestion
Tells `devServer` to write generated assets to the disk.
```

File: src/content/configuration/dev-server.md [hunk 3]
```diff
@@ -1226,3 +1226,36 @@ module.exports = {
...
+```javascript
+module.exports = {
+  //...
+  devServer: {
+    writeToDisk: true
+  }
+};
+```
+
+This option also accepts a Function value, which can be used to filter which files are written to disk. The function follows the same premise as [`Array#filter`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) in which a return value of false will not write the file, and a return value of true will write the file to disk.
```
Comment (EugeneHlushko) [inline src/content/configuration/dev-server.md]:
```suggestion
Providing a `Function` to `devServer.writeToDisk` can be used for filtering. The function follows the same premise as [`Array#filter`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) in which a boolean return value tells if the file should be written to disk.
```

File: src/content/configuration/dev-server.md (hunk 1 above)
Comment (AlexDubok) [reply src/content/configuration/dev-server.md]: Done. thank you

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**